setup.py

```diff
@@ -10,9 +10,9 @@
 # NOTE: These are tested in `continuous_integration/test_imports.sh` If
 # you modify these, make sure to change the corresponding line there.
 extras_require = {
-    "array": ["numpy >= 1.16"],
+    "array": ["numpy >= 1.18"],
     "bag": [],  # keeping for backwards compatibility
-    "dataframe": ["numpy >= 1.16", "pandas >= 0.25.0"],
+    "dataframe": ["numpy >= 1.18", "pandas >= 1.0.0"],
```

Comment:
Idk if we settled on a Pandas version beyond updating it. Went with the next version, which is 1.0.0. Though please let me know if there are concerns with that 🙂

Reply:
It looks like we said [Pandas 1.0.0 in the changelog]( https://github.com/dask/dask/blame/212c1058c19fa543d60d9a294f8f9feda3e1c9dc/docs/source/changelog.rst#L9-L13 ). So maybe we should stick with that

Reply:
Similar comment here about possibly using `1.0` here instead of `1.0.0`